### DISCUSSION TITLE
[Bug] fatal: protocol ''https' is not supported

### DISCUSSION THREAD

**Discussion by LavikLee** [ScoopInstaller/Scoop#5414] [Help and Questions]
## Bug Report
when I update scoop, it can not  update normally, I got this message :

**Updating Scoop...
fatal: protocol ''https' is not supported
Update failed.**
#### Current Behavior

scoop update
### System details

**Windows version:** [e.g. 7, 8, 10, 11]
Windows 10
**OS architecture:** [e.g. 32bit, 64bit, arm64]
64bit
**PowerShell version:** [output of `"$($PSVersionTable.PSVersion)"`]
5
**Additional software:** [(optional) e.g. ConEmu, Git]

#### Scoop Configuration

```json
//# aria2-max-connetion-per-server : 16
aria2-split                    : 16
aria2-min-split-size           : 1M
last_update                    : 2023-02-28T09:10:59.5587986+08:00
scoop_branch                   : master
scoop_repo                     : https://github.com/lukesampson/scoop
```

**Comment by rashil2000** [comment]
Have you checked your network and tried again?

**Reply by LavikLee** [reply]
> Have you checked your network and tried again?

yes, the network works well and I have tried for many times.

**Reply by rashil2000** [reply]
It looks like you've been using Scoop for a long time. What's the git version?

**Reply by LavikLee** [reply]
> It looks like you've been using Scoop for a long time. What's the git version?

**git version 2.39.2.windows.1**

**Reply by LavikLee** [reply]
> It looks like you've been using Scoop for a long time. What's the git version?

Thanks for your help!!

**Reply by LavikLee** [reply]
Finally, I found where the bug is: remote repository address in git config. The address about the remote repository of scoop in ~\scoop\apps\scoop\current\.git\config should be written as https://github.com/lukesampson/scoop, but don't know why, it shows 'https://github.com/lukesampson/scoop' , unnecessary quote mark makes git work failed.  Correct it and scoop works smooth as usual.

**Reply by rashil2000** [reply]
Oh, did you recently change some git configuration in that location? It shouldn't have happened automatically.
